src/components/application_manager/test/commands/mobile/unsubscribe_button_request_test.cc

```diff
@@ -0,0 +1,103 @@
+#include <stdint.h>
+#include <string>
+
+#include "gtest/gtest.h"
+#include "utils/shared_ptr.h"
+#include "commands/command_request_test.h"
+#include "application_manager/mock_application_manager.h"
+#include "application_manager/mock_message_helper.h"
+#include "mobile/unsubscribe_button_request.h"
+
+namespace test {
+namespace components {
+namespace commands_test {
+namespace mobile_commands_test {
+
+namespace am = ::application_manager;
+namespace mobile_result = mobile_apis::Result;
+
+using ::testing::_;
+
+using am::commands::UnsubscribeButtonRequest;
+using am::commands::MessageSharedPtr;
+
+typedef ::utils::SharedPtr<UnsubscribeButtonRequest> CommandPtr;
+
+namespace {
```

Comment:
@LevchenkoS @LuxoftAKutsan I had a discussion with you related to this. @LuxoftAKutsan wrote me to declare constants outside classes to improve readability and was supported by majority of the reviewers. As described in "More Effective C++" anonymous namespace prevents access to constants from other files including improper linking leading to unexpected values of newly declared constants with the same names.